exporter/prometheusexporter/collector.go

```diff
@@ -197,11 +199,33 @@ func (c *collector) convertDoubleHistogram(metric pdata.Metric) (prometheus.Metr
 		points[bucket] = cumCount
 	}
 
+	arrLen := ip.Exemplars().Len()
+	exemplars := make([]prometheus.Exemplar, arrLen)
+	for i := 0; i < arrLen; i++ {
+		e := ip.Exemplars().At(i)
+
+		labels := prometheus.Labels{}
+		e.FilteredAttributes().Range(func(k string, v pdata.Value) bool {
+			labels[k] = v.AsString()
+			return true
+		})
+
+		exemplars[i] = prometheus.Exemplar{
+			Value:     *proto.Float64(e.DoubleVal()),
```

Comment:
This is odd

`proto.Float64` returns a pointer to a float64, which we then dereference to get a `float64`. Can we not just do 
 `Value: e.DoubleVal(),`?

```
// Float64 stores v in a new float64 value and returns a pointer to it.
func Float64(v float64) *float64 { return &v }
```

Reply:
fixed.